specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9093 @@
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Filter": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/FilterProperties"
+            }
+          ]
+        }
+      }
+    },
+    "IntegrationRuntime": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "SelfHosted",
+            "Managed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IntegrationRuntimeType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
+    "IntegrationRuntimeStatus": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "SelfHosted",
+            "Managed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IntegrationRuntimeType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
+    "IntegrationRuntimeAuthKeys": {
+      "type": "object",
+      "properties": {
+        "authKey1": {
+          "type": "string"
+        },
+        "authKey2": {
+          "type": "string"
+        }
+      }
+    },
+    "IntegrationRuntimeRegenerateKeyParameters": {
+      "type": "object",
+      "properties": {
+        "keyName": {
+          "type": "string"
+        }
+      }
+    },
+    "IntegrationRuntimeList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IntegrationRuntime"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "count": {
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "OkResult": {
+      "type": "object",
+      "properties": {
+        "statusCode": {
+          "format": "int32",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "EnableInteractiveQueryForIntegrationRuntimeRequest": {
+      "type": "object",
+      "properties": {
+        "autoTerminationMinutes": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "ManagedVirtualNetworkProperties": {
+      "type": "object",
+      "properties": {
+        "vNetId": {
+          "type": "string",
+          "readOnly": true
+        },
+        "preventDataExfiltration": {
```

Comment:
preventDataExfiltration should not expose to customers

Reply:
suppressed preventDataExfiltration